rbs/SigsRewriter.cc

```diff
@@ -45,151 +45,6 @@ parser::Node *signaturesTarget(parser::Node *node) {
     return nullptr;
 }
 
-Comments signaturesForLoc(core::MutableContext ctx, core::LocOffsets loc) {
```

Comment:
Do we still need `isVisibilitySend`, `isAttrAccessorSend` and `signaturesTarget`?